package.json

```diff
@@ -55,5 +55,8 @@
     "nhsuk-frontend": "^8.0.2",
     "regexpp": "^3.2.0",
     "text-table": "^0.2.0"
+  },
+  "dependencies": {
+    "@nhsbsa/cookie-consent-component": "0.0.6"
```

Comment:
Looks like this dependency needs updating. NOOP for this PR.

Reply:
Also, could add it to dev dependencies to keep dependencies bundles together. Eleventy doesn't care for dependencies or dev-dependencies to my knowledge.

Reply:
latest is 0.0.6?

https://www.npmjs.com/package/@nhsbsa/cookie-consent-component

Reply:
moved to dev dependency